src/pipeline.ml

```diff
@@ -214,5 +218,9 @@ let v ?app ?notify:channel ?filter ~sched ~staging_auth () =
     ];
     ocaml_bench, "sandmark-nightly", [
       docker "Dockerfile" ["main", "ocurrent/sandmark-nightly:live", [`Ci3 "sandmark_sandmark"]]
+    ];
+
+    tarides, "tezos-ci", [
+      docker "Dockerfile" ["live", "ocurrent/tezos-ci-service:live", [`Tezos "tezos-ci_ci"]]
```

Comment:
Pushing to docker hub as `ocurrent/tezos-ci-service` perhaps it should go somewhere else?

Reply:
Pushing to `ocurrent/tezos-ci` for now